pulsar/message.go

```diff
@@ -120,19 +124,65 @@ type Message interface {
 type MessageID interface {
 	// Serialize the message id into a sequence of bytes that can be stored somewhere else
 	Serialize() []byte
+	// String the message id represented as a string
+	String() string
```

Comment:
This is not needed the struct just needs to implement the Stringer interface
https://golang.org/pkg/fmt/#Stringer